tests/test_utils/__init__.py

```diff
@@ -9,11 +9,13 @@
 
 from multiaddr import Multiaddr, protocols
 
+import hivemind
 from hivemind import find_open_port
 from hivemind.p2p.p2p_daemon_bindings.p2pclient import Client
 
 
 TIMEOUT_DURATION = 30  # seconds
+P2PD_PATH = os.path.join(os.path.dirname(hivemind.__file__), 'hivemind_cli', 'p2pd')
```

Comment:
Please use [pkg_resources.resource_filename](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#resource-extraction)

Reply:
I'm not ready to agree that this is better. 
But as you wish.